containers/Home/styles/styles.tsx

```diff
@@ -0,0 +1,240 @@
+import { BackGroundColor, FontColor, GridBreakpoints } from 'style/Theme'
+import styled from 'styled-components'
+import Image from 'next/image'
+
+export const StyledMain = styled.main`
+height: auto;
+width: 100%;
+background-color: ${BackGroundColor.default};
+color: ${FontColor.default};
+margin: 0;
+padding: 10px;
+align-items: center;
+justify-content: center;
+
+@media(max-width: ${GridBreakpoints.lg}){
+  flex-direction: row;
+  display: flex;
+}
+`
+
+export const StyledContainer = styled.div`  
+  min-height: 100vh;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${BackGroundColor.default};
+`
+
+const StyledInfo = styled.div`  
+display: flex;
+flex-direction: row;
+
+@media(max-width: ${GridBreakpoints.lg}) {
+  flex-direction: column;
+}
+`
+
+const StyledWrapperInfo = styled.div`
+display: flex;
+flex-direction: row;
+
+@media(max-width: ${GridBreakpoints.lg}){
+  flex-direction: column
+}
+`
+
+export {  StyledInfo, StyledWrapperInfo
+}
+
+
+export const StyledWrapperHead = styled.div`
+  @media(max-width: ${GridBreakpoints.lg}){
+    width: auto;
+    display: none;
+  }
+`
+
+export const StyledAboutHead = styled.div`
+background-color: ${BackGroundColor.primary};
+display: flex;
+align-items: center;
+justify-content: center;
+padding: 20px;
+flex-direction: column;
+`
+
+export const StyledWrapperImage = styled.div`
+  width: 80px;
+  height: 80px;
+  position: relative;
+/* & ::before{
```

Comment:
In further development, any unused code should be removed. If it is for further use, please add a comment TODO for it